CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No changes yet.
+### Added
+- gRPC: accept keepalive paramters from yaml configurator for the gRPC outbounds.
```

Comment:
nit: typo I paramters
nit: you can also simplify this sentence like `gRPC: accept keepalive parameters for gRPC outbound configuration`